bower.json

```diff
@@ -23,7 +23,7 @@
     "cf-icons": "^1.0.0",
     "cf-pagination": "^1.0.0",
     "cf-typography": "^1.0.0",
-    "cf-layout": "^1.0.0"
+    "cf-layout": "^2.0.0"
```

Comment:
Yeah, but the others that were already over 1.0.0 before the 1.0.0 push have more specific version numbers.